contracts/modules/staking/StakingViewer.sol

```diff
@@ -94,6 +95,10 @@ contract StakingViewer is Multicall {
   function stakingProducts() internal view returns (IStakingProducts) {
     return IStakingProducts(master.contractAddresses('SP'));
   }
+  
+  function coverProducts() internal view returns (ICoverProducts) {
+    return ICoverProducts(master.contractAddresses('CP'));
+  }
```

Comment:
This replaces `cover()` function entirely hence it can take its place

Reply:
yes deleted